docs/evm/ethereum-vs-flow-evm.md

```diff
@@ -0,0 +1,130 @@
+---
+title: Differences between EVM on Flow and Ethereum
+sidebar_label: EVM on Flow vs. Ethereum
+sidebar_position: 14
```

Comment:
Lets put this under 'Why EVM on Flow' and change the position to 2. It is a more logical grouping given content.

Reply:
updated